stl/src/ppltasks.cpp

```diff
@@ -36,15 +36,15 @@ namespace Concurrency {
 
     namespace details {
         [[noreturn]] _CRTIMP2 void __cdecl _ReportUnobservedException() {
-
-#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_CRT_APP)
-            if (IsProcessorFeaturePresent(PF_FASTFAIL_AVAILABLE))
-#endif
-            {
+#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_CRT_APP) && _STL_WIN32_WINNT < _WIN32_WINNT_WIN8
+            if (IsProcessorFeaturePresent(PF_FASTFAIL_AVAILABLE)) {
                 __fastfail(FAST_FAIL_INVALID_ARG);
             }
 
-            std::terminate();
+            std::abort();
+#else // ^^^ __fastfail conditionally available  / __fastfail always available  vvv
+            __fastfail(FAST_FAIL_INVALID_ARG);

```

Comment:
We can avoid this repetition, and the earlier pattern of having two noreturn functions on a control path, by simply inverting the condition.